main.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/0xPolygon/minimal/command"
+	"github.com/0xPolygon/minimal/command/util"
```

Comment:
Feel free to remove the comments at the bottom of this file

Reply:
Removed 👍

Reply:
where can i find the library github.com/0xPolygon/minimal/command/util? i want to check the functions it has so i would appreciate if you could pass the link as i can't find it in go.dev or in the old github repository.

Reply:
> where can i find the library github.com/0xPolygon/minimal/command/util? i want to check the functions it has so i would appreciate if you could pass the link as i can't find it in go.dev or in the old github repository.

Here it is: https://github.com/0xPolygon/polygon-sdk/blob/develop/command/util/commands.go